xarray/core/formatting.py

```diff
@@ -186,9 +186,23 @@ def _summarize_var_or_coord(name, var, col_width, show_values=True,
         values_str = format_array_flat(var, max_width - len(front_str))
     else:
         values_str = u'...'
+
     return front_str + values_str
 
 
+def _summarize_coord_multiindex(coord, col_width, marker):
+    first_col = pretty_print(u'  %s %s ' % (marker, coord.name), col_width)
+    return u'%s(%s) MultiIndex' % (first_col, unicode_type(coord.dims[0]))
+
+
+def _summarize_coord_levels(coord, col_width, marker):
+    # TODO: maybe slicing based on calculated number of displayed values
```

Comment:
It's almost certainly overkill, but you could write:

    max_width = OPTIONS['display_width']
    max_possibly_relevant = max(int(np.ceil(max_width / 2.0)), 1)
    relevant_coord = coord[:max_possibly_relevant]

Either way, you probably do want to slice the coordinate outside the look.